File: spring-grpc-server/server/src/main/java/com/example/springgrpc/server/GrpcClientProperties.java
package com.example.springgrpc.server;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties("sample.grpc.client")
public class GrpcClientProperties {
    private String host = "localhost";
    private int port = 9090;
    private long timeoutSeconds = 5;

    public String getHost() {
        return host;
    }

    public void setHost(String host) {
        this.host = host;
    }

    public int getPort() {
        return port;
    }

    public void setPort(int port) {
        this.port = port;
    }

    public long getTimeoutSeconds() {
        return timeoutSeconds;
    }

    public void setTimeoutSeconds(long timeoutSeconds) {
        this.timeoutSeconds = timeoutSeconds;
    }
}

File: spring-grpc-server-kotlin/server/src/main/kotlin/com/example/springgrpc/server/GrpcClientProperties.kt
package com.example.springgrpc.server

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties("sample.grpc.client")
data class GrpcClientProperties(
    val host: String = "localhost",
    val port: Int = 9090,
    val timeoutSeconds: Long = 5,
)
